doc/cephfs/fs-volumes.rst

```diff
@@ -260,6 +260,24 @@ List subvolumes using::
 
 .. note:: subvolumes that are removed but have snapshots retained, are also listed.
 
+Set the user metadata in the key-value format for a subvolume using::
+
```

Comment:
Maybe say that key and value can be of arbitrary length? We don't seem to enforce any length restriction.

What about the type of key and value? Looks like we enforce type restriction on both the key and value. It should be a string of ASCII characters as specified in python's string.printable.https://docs.python.org/3/library/string.html#string.printable
(I guess value can be an empty string?)

See, https://github.com/ceph/ceph/blob/master/src/pybind/ceph_argparse.py#L338

Reply:
@ajarr Updated docs and added above suggested points.